libs/sdk-ui-ext/src/internal/components/configurationControls/clustering/NumberOfClustersControl.tsx

```diff
@@ -0,0 +1,48 @@
+// (C) 2019-2024 GoodData Corporation
+import React from "react";
+import InputControl from "../InputControl.js";
+import { IVisualizationProperties } from "../../../interfaces/Visualization.js";
+import { messages } from "../../../../locales.js";
+
+export interface INumberOfClustersControlProps {
+    valuePath: string;
+    disabled: boolean;
+    properties: IVisualizationProperties;
+    pushData: (data: any) => any;
+}
+
+const validPositiveNumberHigherThanZero = /^[1-9][0-9]*$/;
+
+export const NumberOfClustersControl = ({
+    disabled,
+    valuePath,
+    properties,
+    pushData,
+}: INumberOfClustersControlProps) => {
+    properties?.controls?.clustering?.numberOfClusters ?? "3";
```

Comment:
Forgotten variable assignment for later use? Maybe also separate "3" as a constant as it is used multiple times here.

Reply:
I can also see number 3 in other files, but used as a number, not as string. Can we reuse it everywhere as the same constant or is it something different?